website/templates/athleteCoachView.html

```diff
@@ -5,6 +5,13 @@
 
 <h1>{{ athlete.first_name }} {{ athlete.last_name }}</h1>
 </br>
+
```

Comment:
Do we want the coach to be able to download the Athlete's data?

Reply:
good point, deleted the csv links from coach template